recipes/python-toolchain/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: python-toolchain
+  version: 1.0.0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - setuptools
+    - pip
+
+test:
+  commands:
+    - test -f "${PREFIX}/etc/pip.conf"                                 # [unix]
+   #- test -f "${STDLIB_DIR}/distutils/distutils.cfg"                  # [unix]
+    - if not exist "%LIBRARY_PREFIX%\\etc\\pip.conf" exit 1            # [win]
+   #- if not exist "%STDLIB_DIR%\\distutils\\distutils.cfg" exit 1     # [win]
```

Comment:
Sorry, I confused the two issues. Also, using `activate` to solve this problem on *NIXes. This is the `conda-build` issue ( https://github.com/conda/conda-build/issues/910 ) that pertains to fixing this problem.